src/goSuggest.ts

```diff
@@ -75,6 +81,44 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 		});
 	}
 
+	public resolveCompletionItem(item: vscode.CompletionItem, token: vscode.CancellationToken): vscode.ProviderResult<vscode.CompletionItem> {
+		if (item instanceof ExtendedCompletionItem) {
+			const goRuntimePath = getBinPath('go');
+			if (!goRuntimePath) {
+				vscode.window.showInformationMessage('Cannot find "go" binary. Update PATH or GOROOT appropriately');
+				return;
```

Comment:
I am not sure if the completion feature fails in some way when we return this way. Can you test this scenario? One way to be sure is to always return the `item` as is when we are not able to get the docs.

Reply:
If you return with null/undefined, it simply uses the original item. It's documented here: [CompletionItemProvider](https://code.visualstudio.com/docs/extensionAPI/vscode-api#CompletionItemProvider.resolveCompletionItem)